packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -9,18 +10,22 @@ interface Copy {
 interface ClassName {
   container?: string;
   header?: string;
+  title?: string;
 }
 export interface FormWrapperProps {
   children: ReactNode;
-  className?: string | ClassName;
```

Comment:
Had to remove the union type. Not really a fan in the first place, but for each and every new property to the `ClassName` object, we will add another line of ternary operator.